bitnami/postgresql-repmgr/11/debian-11/rootfs/opt/bitnami/scripts/librepmgr.sh

```diff
@@ -627,7 +632,24 @@ repmgr_clone_primary() {
 #########################
 repmgr_pgrewind() {
     info "Running pg_rewind data to primary node..."
-    local -r flags=("-D" "$POSTGRESQL_DATA_DIR" "--source-server" "host=${REPMGR_CURRENT_PRIMARY_HOST} port=${REPMGR_CURRENT_PRIMARY_PORT} user=${REPMGR_USERNAME} dbname=${REPMGR_DATABASE}")
+
+    postgresql_create_pghba
+    postgresql_copy_custom_configuration
```

Comment:
Why do you need this piece of code?  I think is not needed, the function `repmgr_postgresql_configuration` is called from `setup.sh` script before running replication manager initialization.

Reply:
When omitted `pg_ctl start` gives an error:
```
LOG:  could not open configuration file "/opt/bitnami/postgresql/conf/pg_hba.conf": No such file or directory
```